### PR TITLE
Fix a typo in the jakartaee-microprofile-example readme file

### PR DESCRIPTION
**Pull request by anunnakian**
This PR is about to fix a typo in the readme file of the `jakartaee-microprofile-example` folder. 
It'll be great if there is no typo at all 😄

### CODE REVIEW

File: jakartaee-microprofile-example/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ export HUGGING_FACE_API_KEY=<your Hugging Face read token>
 Use the Maven wrapper to start the application by using the [Liberty dev mode](https://openliberty.io/docs/latest/development-mode.html):
 
 ```
-./mvnw liberty:dev
+c
```
Comment (langchain4j) [inline jakartaee-microprofile-example/README.md:33]: is this change intended?

**Review by langchain4j — COMMENTED** [review]
HI @anunnakian thanks!

File: jakartaee-microprofile-example/README.md (hunk 1 above)
Comment (anunnakian) [reply jakartaee-microprofile-example/README.md]: Not at all! I revert it

**Review by langchain4j — APPROVED** [review]
@anunnakian thanks!
